Libraries/MLXVLM/Models/Idefics3.swift

```diff
@@ -663,9 +672,12 @@ public class Idefics3: Module, VLMModel, KVCacheDimensionProvider {
         return final
     }
 
+    // inputs_merger
+    // TODO: why did we need to do changes here? Do we need a new modelling class, or did this never work (for tiling)?
```

Comment:
This is actually a pending to-do for Idefics3. We can remove the comment here, but revisit whether this works for the previous smolvlm.